### PR TITLE
[CM-358] Add the form data post #trivial 

### PR DESCRIPTION
**Pull request by Sunilkumarr**
## Summary
* Added post form data 
* Added hidden fields 
 
Note: Validation of regex is not there 

## Testing
Tried sending the form data to dialog flow and data is received

### CODE REVIEW

File: Sources/Views/ALKFormCell.swift [hunk 1]
```diff
@@ -151,3 +190,15 @@ class NestedCellTableView: UITableView {
...
+
+class FormDataSubmit {
+    var listOfUITextFieldWithPos : [Int: UITextField]?
+    var singleSelect : [Int : Int]?
+    var multiSelect : [Int : [Int]]?
+
+    init() {
+        listOfUITextFieldWithPos = [Int: UITextField]()
+        singleSelect = [Int : Int]()
+        multiSelect = [Int : [Int]]()
```
Comment (mukeshthawani) [inline Sources/Views/ALKFormCell.swift:202]: We can set this as default values. Also, change it to `Struct` and remove the `init`(as default initializer is there).

File: Sources/Views/ALKFormCell.swift [hunk 2]
```diff
@@ -151,3 +190,15 @@ class NestedCellTableView: UITableView {
         }
     }
 }
+
+class FormDataSubmit {
+    var listOfUITextFieldWithPos : [Int: UITextField]?
```
Comment (mukeshthawani) [inline Sources/Views/ALKFormCell.swift:195]: It should be just `textFields` and same for other properties. And this should just have the text in value.

File: Sources/Views/ALKFormCell.swift [hunk 3]
```diff
@@ -113,6 +125,31 @@ extension ALKFormCell: UITableViewDataSource, UITableViewDelegate {
...
             let cell: ALKFormMultiSelectItemCell = tableView.dequeueReusableCell(forIndexPath: indexPath)
+            cell.cellSelected = {
+                guard var multiSelect = self.formDataSubmit.multiSelect else {
+                    return
+                }
+
+                if let array = multiSelect[indexPath.section] {
+                    var newArray = array
+                    if array.contains(indexPath.row) {
+                        newArray.remove(at: indexPath.row)
```
Comment (mukeshthawani) [inline Sources/Views/ALKFormCell.swift:136]: Current row may not be present at the same index as we are just appending values in this array.

Comment (Sunilkumarr) [reply Sources/Views/ALKFormCell.swift:129]: done

File: Sources/Views/ALKFormCell.swift (hunk 2 above)
Comment (Sunilkumarr) [reply Sources/Views/ALKFormCell.swift]: done

File: Sources/Views/ALKFormCell.swift (hunk 1 above)
Comment (Sunilkumarr) [reply Sources/Views/ALKFormCell.swift]: done

**Pull request merged**